Cargo.toml

```
@@ -30,37 +36,31 @@ debug = true
 
 [dependencies]
 fixedbitset = { version = "0.4.0", default-features = false }
-quickcheck = { optional = true, version = "0.8", default-features = false }
 indexmap = { version = "1.6.2" }
+quickcheck = { optional = true, version = "0.8", default-features = false }
 serde = { version = "1.0", optional = true }
 serde_derive = { version = "1.0", optional = true }
 
 [dev-dependencies]
-rand = "0.5.5"
-odds = { version = "0.4.0" }
+bincode = "1.3.3"
 defmac = "0.2.1"
 itertools = { version = "0.10.1", default-features = false }
-bincode = "1.3.3"
+odds = { version = "0.4.0" }
+rand = "0.5.5"
 
 [features]
+
+# feature flags for testing use only
+all = ["unstable", "quickcheck", "matrix_graph", "stable_graph", "graphmap"]
 default = ["graphmap", "stable_graph", "matrix_graph"]
-graphmap = []
-serde-1 = ["serde", "serde_derive"]
-stable_graph = []
-matrix_graph = []
 
 # For unstable features
 generate = []
```

I'd move the comment here.
```suggestion
generate = [] # For unstable features
```

---

Fixed it :)